### ISSUE TITLE
CRUD actions return with fresh record

### ISSUE THREAD

**Issue by vinicius73** [codecasts/spa-starter-kit#94]
API actions like [webservice/app/Http/Controllers/CategoriesController.php#L43](https://github.com/codecasts/spa-starter-kit/blob/24b03a1e772191ec55a3227ea13d56ab3f46b852/webservice/app/Http/Controllers/CategoriesController.php#L43) returna only a success message

```php
Category::create($request->only('name'));

return $this->response->json([
   'result' => 'success',
]);
```

But will be more useful if the response contains the new record's data

**Comment by flyingluscas** [comment]
@vinicius73 I was thinking about that too, and also about return the right status codes, like when creating something return a 201 status code instead a 200, I can implement that :)

**Comment by vedovelli** [comment]
Closed by #97
